### ISSUE TITLE
juliac.jl currently needs to be run from static-julia directory

### ISSUE THREAD

**Issue by tkoolen** [JuliaComputing/static-julia#1]
Otherwise you get something like
```
twan@tk-loco:~/code/RigidBodyDynamics/v0.6/RigidBodyDynamics$ julia ~/code/static-julia/juliac.jl hello.jl /opt/julia-0.6/ $JULIA_PKGDIR 
Program File : hello.jl
Julia Install Path: /opt/julia-0.6/
gcc: error: program.c: No such file or directory
ERROR: LoadError: failed process: Process(`gcc -m64 program.c -o hello libhello.so -std=gnu99 -I/opt/julia-0.6/include/julia -DJULIA_ENABLE_THREADING=1 -fPIC -L/opt/julia-0.6/lib -Wl,--export-dynamic -Wl,-rpath,/opt/julia-0.6/lib -Wl,-rpath,/opt/julia-0.6/lib/julia -ljulia -lm '-Wl,-rpath,$ORIGIN'`, ProcessExited(1)) [1]
Stacktrace:
 [1] pipeline_error(::Base.Process) at ./process.jl:682
 [2] run(::Cmd) at ./process.jl:651
 [3] compile(::String, ::String, ::String) at /home/twan/code/static-julia/juliac.jl:35
 [4] include_from_node1(::String) at ./loading.jl:569
 [5] include(::String) at ./sysimg.jl:14
 [6] process_options(::Base.JLOptions) at ./client.jl:305
 [7] _start() at ./client.jl:371
while loading /home/twan/code/static-julia/juliac.jl, in expression starting on line 51
```
(the library is generated just fine)

**Comment by tkoolen** [comment]
Note that the issue is still valid; the readme has just been updated.

**Comment by ViralBShah** [comment]
@lucatrv Can we close this one?

**Comment by lucatrv** [comment]
Yes this was fixed with PR #15
